lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1501,6 +1529,7 @@ class SemanticsObject {
       PrimaryRole.image => ImageRoleManager(this),
       PrimaryRole.platformView => PlatformViewRoleManager(this),
       PrimaryRole.generic => GenericRole(this),
+      PrimaryRole.heading => Heading(this),
```

Comment:
nit: let's keep the `generic` role last to underscore that it's the last resort when a more specific role wasn't found.

Reply:
Right, done.